### PR TITLE
Accessibility Report

### PR DESCRIPTION
**Pull request by MoRaad97**
Accessibility Report

- Page titles - no issues found.
- Image text alternatives. "check the comments"
- Text headings line - no issues found.
- Color contrast - "Check Comments"
- Resize - "Check Comments"
- Interaction - no issues found
- Moving content - no issues found
- Multimedia - no issues found
- The basic structure of the page - no issue found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,203 +13,222 @@
...
+        <h4 class="social-media-intro">LET’S CONNECT</h4>
 
-      <nav class="navbarTwo">
-        <ul class="navbarTwo-details">
-          <li class="navbarTwo-list">
-            <a class="navbarTwo-link" href="#portfolio">Portfolio</a>
+        <ul class="social-media-icon">
+          <li class="social-media">
+            <a class="list-style" href="#"
+              ><img src="tweeter.svg" alt="twitter"
```
Comment (MoRaad97) [inline index.html:56]: Image text alternatives Fix

File: index.html [hunk 2]
```diff
@@ -13,203 +13,222 @@
...
+            /></a>
           </li>
-          <li class="navbarTwo-list">
-            <a class="navbarTwo-link" href="#about">About</a>
+          <li class="social-media">
+            <a class="list-style" href="#"
+              ><img src="Linkedin-icon.svg" alt="Linkedin"
```
Comment (MoRaad97) [inline index.html:61]: Image text alternatives Fix

File: index.html [hunk 3]
```diff
@@ -13,203 +13,222 @@
...
+            /></a>
+          </li>
+          <li class="social-media">
+            <a class="list-style" href="#"
+              ><img src="mediaThree.svg" alt="mediaThree"
```
Comment (MoRaad97) [inline index.html:66]: Image text alternatives Fix

File: index.html [hunk 4]
```diff
@@ -13,203 +13,222 @@
...
+            /></a>
           </li>
-          <li class="navbarTwo-list">
-            <a class="navbarTwo-link" href="#contact">Contact</a>
+          <li class="social-media">
+            <a class="list-style" href="#"
+              ><img src="mediaFour.svg" alt="mediaFour"
```
Comment (MoRaad97) [inline index.html:71]: Image text alternatives Fix

File: index.html [hunk 5]
```diff
@@ -13,203 +13,222 @@
...
+            /></a>
+          </li>
+          <li class="social-media">
+            <a class="list-style" href="#"
+              ><img src="mediaFive.svg" alt="mediaFive"
```
Comment (MoRaad97) [inline index.html:76]: Image text alternatives Fix

File: index.html [hunk 6]
```diff
@@ -13,203 +13,222 @@
...
-      
-      <div class="card first-card">
-       <div class="card-img-head" ><img src="desk-v-card-one.svg" alt="" class="card-img" /> </div>
 
-        
+      <div class="card first-card">
+        <div class="card-img-head">
+          <img
+            src="desk-v-card-one.svg"
+            alt="desk-v-card-one"
```
Comment (MoRaad97) [inline index.html:90]: Image text alternatives Fix

File: index.html [hunk 7]
```diff
@@ -13,203 +13,222 @@
...
-        <button type="submit" class="btnNormal">See Project</button>
-
-           
-      </div>
+          <h2 class="project-title">Tonic</h2>
+
+          <ul class="workInfo">
+            <li class="workInfoDetails first-work">
+              <a class="workInfoDetailsLink firstWorkLink" href="#">CANOPY</a>
+              <img class="counter" src="Counter.svg" alt="Counter" />
```
Comment (MoRaad97) [inline index.html:101]: Image text alternatives Fix

File: index.html [hunk 8]
```diff
@@ -13,203 +13,222 @@
...
+            </li>
+            <li class="workInfoDetails">
+              <a class="workInfoDetailsLink" href="#">Back End Dev</a>
+              <img class="counter" src="Counter.svg" alt="Counter" />
```
Comment (MoRaad97) [inline index.html:105]: Image text alternatives Fix

File: index.html [hunk 9]
```diff
@@ -13,203 +13,222 @@
...
+          <h2 class="project-title">Multi-Post Stories</h2>
+
+          <ul class="workInfo">
+            <li class="workInfoDetails first-work">
+              <a class="workInfoDetailsLink firstWorkLink" href="#">CANOPY</a>
+              <img class="counter" src="Counter.svg" alt="Counter" />
+            </li>
+            <li class="workInfoDetails">
+              <a class="workInfoDetailsLink" href="#">Back End Dev</a>
+              <img class="counter" src="Counter.svg" alt="Counter" />
```
Comment (MoRaad97) [inline index.html:156]: Image text alternatives Fix

File: index.html [hunk 10]
```diff
@@ -13,203 +13,222 @@
...
-        </ul>
-
-        <button type="button" class="btnNormal">See Project</button>
-      </div>
+          <h2 class="project-title">Multi-Post Stories</h2>
+
+          <ul class="workInfo">
+            <li class="workInfoDetails first-work">
+              <a class="workInfoDetailsLink firstWorkLink" href="#">CANOPY</a>
+              <img class="counter" src="Counter.svg" alt="Counter" />
```
Comment (MoRaad97) [inline index.html:152]: Image text alternatives Fix

File: index.html [hunk 11]
```diff
@@ -13,203 +13,222 @@
...
       <!-- card ends here -->
 
       <!-- single card start here -->
 
       <div class="card third-card">
-        <div class="card-img-head" > <img src="card-three-desktop.svg" alt="" class="card-img" /> </div>
+        <div class="card-img-head">
+          <img
+            src="card-three-desktop.svg"
+            alt="card-three-desktop"
```
Comment (MoRaad97) [inline index.html:192]: Image text alternatives Fix

File: index.html [hunk 12]
```diff
@@ -13,203 +13,222 @@
...
+          <h2 class="project-title">Tonic</h2>
+
+          <ul class="workInfo">
+            <li class="workInfoDetails first-work">
+              <a class="workInfoDetailsLink firstWorkLink" href="#">CANOPY</a>
+              <img class="counter" src="Counter.svg" alt="Counter" />
+            </li>
+            <li class="workInfoDetails">
+              <a class="workInfoDetailsLink" href="#">Back End Dev</a>
+              <img class="counter" src="Counter.svg" alt="Counter" />
```
Comment (MoRaad97) [inline index.html:207]: Image text alternatives Fix

File: index.html [hunk 13]
```diff
@@ -218,207 +237,230 @@ <h2 class="project-title">Tonic</h2>
       <!-- single card start here -->
 
       <div class="card fouth-card">
-        <div class="card-img-head" > <img src="card-four-desktop.svg" alt="" class="card-img" /> </div>
+        <div class="card-img-head">
+          <img
+            src="card-four-desktop.svg"
+            alt="card-four-desktop"
+            class="card-img"
```
Comment (MoRaad97) [inline index.html:244]: Image text alternatives Fix

File: index.html [hunk 14]
```diff
@@ -218,207 +237,230 @@ <h2 class="project-title">Tonic</h2>
...
+          <h2 class="project-title">Multi-Post Stories</h2>
+
+          <ul class="workInfo">
+            <li class="workInfoDetails first-work">
+              <a class="workInfoDetailsLink firstWorkLink" href="#">CANOPY</a>
+              <img class="counter" src="Counter.svg" alt="Counter" />
+            </li>
+            <li class="workInfoDetails">
+              <a class="workInfoDetailsLink" href="#">Back End Dev</a>
+              <img class="counter" src="Counter.svg" alt="Counter" />
```
Comment (MoRaad97) [inline index.html:258]: Image text alternatives Fix

File: index.html [hunk 15]
```diff
@@ -218,207 +237,230 @@ <h2 class="project-title">Tonic</h2>
...
+              /></a>
+            </li>
+            <li class="social-media">
+              <a class="list-style" href="#"
+                ><img src="mediaFour.svg" alt="mediaFour"
+              /></a>
+            </li>
+            <li class="social-media">
+              <a class="list-style" href="#"
+                ><img src="mediaFive.svg" alt="mediaFive"
```
Comment (MoRaad97) [inline index.html:331]: Image text alternatives Fix

File: index.html [hunk 16]
```diff
@@ -218,207 +237,230 @@ <h2 class="project-title">Tonic</h2>
...
+                  alt="javascriptImage"
+                />
+
+                <h4 class="lang-icon-">JavaScript</h4>
+              </li>
+              <li class="langIconDetails sec-lang">
+                <img
+                  class="iconDetailsImg"
+                  src="htmlImage.svg"
+                  alt="htmlImage"
```
Comment (MoRaad97) [inline index.html:368]: Image text alternatives Fix

File: index.html [hunk 17]
```diff
@@ -218,207 +237,230 @@ <h2 class="project-title">Tonic</h2>
...
+                src="skillimage.svg"
+                alt="language-display"
+              />
+            </li>
+          </ul>
+
+          <img
+            src="Separator-Bottom.svg"
+            class="sept-btn"
+            alt="Separator-Bottom"
```
Comment (MoRaad97) [inline index.html:412]: Image text alternatives Fix

**Review by MoRaad97 — COMMENTED** [review]
## Image text alternatives Fix

File: style.css [hunk 18]
```diff
@@ -39,7 +39,7 @@ body {
 }
 
 .social-media-intro {
-  color: #7f8cff;
+  color: hsl(233.9,100%,40.9%);
```
Comment (henrycode460) [inline style.css:42]: Improved text contrast to meet accessibility standard

File: style.css [hunk 19]
```diff
@@ -92,6 +92,7 @@ body {
 .card-img {
   width: 100%;
   height: auto;
+  filter :brightness(0.7);
```
Comment (henrycode460) [inline style.css:95]: Improved card images contrast to meet accessibility standard

File: style.css [hunk 20]
```diff
@@ -198,11 +199,11 @@ body {
 
 .btnNormal {
   background: #fff;
-  color: #7f8cff;
+  color: hsl(233.9,100%,20.9%);
```
Comment (henrycode460) [inline style.css:202]: Improved button text color to meet accessibility standard

**Review by aimemalaika — APPROVED** [review]
Hi @henrycode460, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**